docs/Data/Injector.md

```diff
@@ -0,0 +1,52 @@
+## Module Data.Injector
+
+#### `Injector`
+
+``` purescript
+data Injector a b
+  = Injector (a -> b) (b -> Maybe a)
```

Comment:
Since I don't think I have push access to your fork, here's a patch that you can apply: https://gist.github.com/jonsterling/21dc46bc4dff553878ea